src/js/components/Span/TimeButton.tsx

```diff
@@ -87,25 +42,12 @@ export default function TimeButton({
 
   if (editing) return <TimeInput timeArg={timeArg} onSubmit={onSubmit} />
   return (
-    <div
-      className={classNames("time-picker-button", {
-        hovering: fuzzy.hovering
-      })}
-      onMouseLeave={fuzzy.mouseLeave}
-      onClick={onClick}
-    >
-      <div className="hover-zone" />
-      <TimeSteppers
```

Comment:
These time steppers were not being used anymore. Along with all the mouse handling above.